mslib/msui/_tests/test_mss_pyui.py

```diff
@@ -29,13 +29,34 @@
 import sys
 import mock
 import os
+from urllib.request import urlopen
+import re
 from PyQt5 import QtWidgets, QtTest
+from mslib import __version__
 from mslib._tests.constants import ROOT_DIR
 import mslib.msui.mss_pyui as mss_pyui
 from mslib.plugins.io.text import load_from_txt, save_to_txt
 from mslib.plugins.io.flitestar import load_from_flitestar
 
 
+class Test_MSS_AboutDialog():
+    def setup(self):
+        self.application = QtWidgets.QApplication(sys.argv)
+        self.window = mss_pyui.MSS_AboutDialog()
+
+    def test_milestone_url(self):
+        with urlopen(self.window.milestone_url) as f:
+            text = f.read()
+        pattern = f'value="is:closed milestone:{__version__[:-1]}"'
+        assert re.search(pattern, text.decode('utf-8'))
```

Comment:
The pattern doesn't seem to contain any special regex.
In which case, is there a reason not to use 
`assert pattern in text.decode('utf-8')`
instead of
`assert re.search(pattern, text.decode('utf-8'))`
?

Reply:
Also, `.` is interpreted as "any character" unless escaped. 
So while it does work for our case, it would also match versions like "31022" instead of "3.0.2"

Reply:
oh, the first approach is much better, and I completly forgot about the "." feature.